src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/CertificateAuthority.cs

```diff
@@ -1040,10 +1040,12 @@ internal static KeyFactory[] BuildVariantFactories()
             {
                 List<KeyFactory> factories = [RSA, ECDsa];
 
+#if !WINDOWS // MLDsa certificate support on Windows is not available yet. Remove this once it is.
```

Comment:
This needs to be a runtime check, not compile-time.  Not all test assemblies that include this are necessarily split-compile.